### PR TITLE
doc how to use from Akka classic

### PR DESCRIPTION
**Pull request by patriknw**
Request from PM

### CODE REVIEW

File: docs/src/main/paradox/classic.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Scala
+:  @@snip [ClassicDocExample.scala](/examples/src/test/scala/docs/classic/ClassicDocExample.scala) { #system }
+
+Java
+:  @@snip [ClassicDocExample.java](/examples/src/test/java/jdocs/classic/ClassicDocExample.java) { #import-adapter #system }
+
+## PersistentActor
+
+@ref:[Events from Akka Persistence](eventsourced.md) can be emitted from `PersistentActor` and consumed by a
```
Comment (seglo) [inline docs/src/main/paradox/classic.md:20]:
```suggestion
@ref:[Events from Akka Classic Persistence](eventsourced.md) can be emitted from `PersistentActor` and consumed by a
```

File: docs/src/main/paradox/classic.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Akka Classic
+
+Akka Projections can be used with the [new Actor API](https://doc.akka.io/docs/akka/current/typed/actors.html) or
+the [classic Actor API](https://doc.akka.io/docs/akka/current/index-classic.html). The documentation samples
+show the new Actor API, and this page highlights how to use it with the classic Actor API.
+
+## Actor System
+
+The `ActorSystem` is a parameter in several places of the Projections API. That is the `akka.actor.typed.ActorSystem`.
```
Comment (seglo) [inline docs/src/main/paradox/classic.md:9]: Would it make sense to add apidocs to the classic and typed ActorSystems?

Comment (patriknw) [reply docs/src/main/paradox/classic.md:9]: The important part is to show the package name, which I don't think apidoc links show.

Comment (seglo) [reply docs/src/main/paradox/classic.md:9]: I think you're right. :+1:

**Pull request merged**